src/panel/spoiler_card/SpoilerCardContainer.js

```diff
@@ -64,6 +65,7 @@ class SpoilerCardContainer extends React.Component {
 
 	handleSaveTags() {
 		spoilerCardActions.editTags(this.props.index, this.title, this.tags);
+		this.tags = this.tags.trim();
 	}
```

Comment:
Yeah but here they haven't been trimmed, as that happened when they were converted to an array